plugins/steelseries/steelseries.quirk

```diff
@@ -12,6 +12,13 @@ GType = FuSteelseriesSonic
 Icon = input-mouse
 
 # Aerox 3 Wireless
+[HIDRAW\VEN_0111&DEV_183A]
+Plugin = steelseries
+GType = FuSteelseriesHid
+Name = Aerox 3 Wireless Mouse
+Icon = input-mouse
+CounterpartGuid = USB\VID_1038&PID_1838
```

Comment:
@hughsie: Could we `_attach()/_detach()` a device using a counterpart device with a different `GType`?